components/ProfilePage/Desktop/PFPInformation.tsx

```diff
@@ -1,17 +1,52 @@
 import { useUserProvider } from "../../../providers/UserProvider"
+import AttributeCard from "./AttributeCard"
 
 const PFPInformation = () => {
-  const { userInfo } = useUserProvider()
+  const { userInfo, metaData } = useUserProvider()
 
   return (
-    <div
-      className="font-quicksand uppercase text-[68px] [writing-mode:vertical-rl]
-            text-black
-            leading-[103.3%]
-            rotate-[180deg]"
-    >
-      {userInfo?.cre8orNumber ? `CRE8OR #${userInfo?.cre8orNumber}` : ""}
-    </div>
+    <>
+      <div
+        className="font-quicksand uppercase text-[68px] [writing-mode:vertical-rl]
+              text-white
+              leading-[103.3%]
+              rotate-[180deg]"
+      >
+        {userInfo?.cre8orNumber ? `CRE8OR #${userInfo?.cre8orNumber}` : ""}
+      </div>
+      <div className="flex flex-col gap-y-[10px]">
```

Comment:
you could just use map. Something like
```
metadata?.attributes?.map((attr) => (
<AttributeCard
 key={attr.trait_type}
label={attr.trait_type}
attribute={attr.value}
))
```
this way you dont have to do all this filtering.

Reply:
In that case, figma desing attributes order is also problem for me. 🤔

Reply:
Does it have to be specific order?

Reply:
I believe there is a meaningful order in the figma design attributes.

Reply:
updated 👍

Reply:
looks like this comment can be `resolved` right?